app/assets/stylesheets/govuk_publishing_components/components/_autocomplete.scss

```diff
@@ -0,0 +1,49 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.gem-c-autocomplete {
+  position: relative;
+
+  > div {
```

Comment:
We try to avoid styling elements - is there a class name you could create for this?